lib/src/ui/input_field_generator.dart

```diff
@@ -56,11 +56,13 @@ class _InputFieldGeneratorState extends State<InputFieldGenerator> {
                   var isLastSetting = i == setting.settings!.length - 1;
                   setting.settings![i].isLastSettingInGroup = isLastSetting;
                   setting.settings![i].partOfGroup = true;
-                  var currentSetting = setting.settings![i];
 
-                  return _buildInputFieldFromControlSetting(
```

Comment:
Assuming this was recursive for pages in pages. Why is this removed?

Reply:
It is still recursive, except it uses the class rather than the method.